src/containers/my-quizzes/selectable-question-quiz-view/SelectableQuestionQuizView.tsx

```diff
@@ -28,13 +37,48 @@ const SelectableQuestionQuizView: FC<SelectableQuestionQuizViewProps> = ({
     )
   })
 
+  const onNext = () => {
+    if (!isLastQuestion) {
+      setSelectedIndex(selectedIndex + 1)
+    }
+  }
+
+  const onBack = () => {
+    if (!isFirstQuestion) {
+      setSelectedIndex(selectedIndex - 1)
+    }
+  }
+
   return (
     <Box>
       <Box sx={styles.selectableList}>{questionsNumberList}</Box>
       <SelectableQuestion
         question={questions[selectedIndex]}
         selectedIndex={selectedIndex}
       />
+      {!isSingleQuestion && (
+        <Box sx={styles.buttons}>
+          {!isFirstQuestion && (
+            <AppButton
+              onClick={onBack}
+              size={SizeEnum.ExtraLarge}
+              variant={ButtonVariantEnum.Tonal}
+            >
+              <ArrowBackIcon fontSize={SizeEnum.Medium} sx={{ mr: '5px' }} />
```

Comment:
We trying to stick all our styles inside `component.styles.ts` file and not to use inline styles.